src/OIDCCryptUtils.purs

```diff
@@ -0,0 +1,48 @@
+module OIDCCryptUtils (RSASIGNTIME(), hashNonce, bindState, unbindState, verifyIdToken) where
```

Comment:
I agree. 

Would a record or type synonyms be okay? I don't like having too many newtypes